src/zkp/pifac.rs

```diff
@@ -6,7 +6,16 @@
 // License, Version 2.0 found in the LICENSE-APACHE file in the root directory
 // of this source tree.
 
-//! Implements the ZKP from Figure 28 of <https://eprint.iacr.org/2021/060.pdf>
+//! Implements a zero-knowledge proof that the modulus N can be factored into
+//! two numbers greater than `2^ℓ` for a parameter ell where `ℓ` is
+//! [`parameters::ELL`](crate::parameters::ELL).
```

Comment:
todo: `N` should be in backticks.

I think it's confusing to use both ℓ and the word "ell". I would rephrase to:
> ...two numbers greater than `2^ℓ`, where `ℓ` is a fixed parameter defined by
//! [`parameters::ELL`](crate::parameters::ELL).

Reply:
I still see a `...` and no link to `parameters::ELL`.